BadgeUpClient/ResourceClients/ProgressClient.cs

```diff
@@ -20,11 +20,24 @@ public ProgressClient(BadgeUpHttpClient httpClient)
 		/// Retrieves a list of achievements and the completion a subject has towards the achievements. The returned list will also contain earned achievement records when present.
 		/// </summary>
 		/// <param name="subjectId">Unique subject the progress is associated with</param>
+		/// <param name="includeAchievements">Include related achievements</param>
+		/// <param name="includeCriteria">Include related criteria</param>
+		/// <param name="includeAwards">Include related awards</param>
 		/// <returns></returns>
-		public async Task<List<Progress>> GetProgress(string subjectId)
+		public async Task<List<Progress>> GetProgress(string subjectId, bool includeAchievements = false, bool includeCriteria = false, bool includeAwards = false)
```

Comment:
Do you think it would make more sense to bundle these options into an object?

Reply:
No, not really. These parameters are optional - so the user doesn't have to specify all of them, and he can take advantage of using named parameters. Like `client.GetAll(id, includeAwards: true);` If there were no optional\named parameters - I would bundle these options into a single object.